README.md

```diff
@@ -13,7 +13,7 @@ Support for additional data sources can be contributed to the [Sematext Agent re
 Sematext Agent uses Influx Line Protocol to ship the collected metrics via HTTPS.  Thus, the collected data can be sent to [Sematext Cloud](https://sematext.com/cloud) (or [Sematext Enterprise](https://sematext.com/enterprise)), or your own InfluxDB.
 
 # Contributing
-Adding a new integration that collects data from one of the available sources is easy and involves no coding.  To add a new integration only a set of configuration (YML) files is needed.  These config files specify the data source, as well as individual metrics to collect.  For more information see [Adding New Sematext Agent Integrations](CONTRIBUTING.md).
+Adding a new integration that collects data from one of the available sources is easy and involves no coding.  To add a new integration only a set of [configuration](https://github.com/sematext/sematext-agent-java/blob/master/docs/metrics-yaml-format.md) (YAML) files is needed.  These config files specify the data source, as well as individual metrics to collect.  For more information see [Adding New Sematext Agent Integrations](CONTRIBUTING.md).
```

Comment:
If YAML files live in the integrations repo, why is the docs about their format in the agent repo?